deploy/runtools/run_farm.py

```diff
@@ -384,14 +384,15 @@ def instance_logger(self, logstr):
     def get_and_install_aws_fpga_sdk(self):
         """ Installs the aws-sdk. This gets us access to tools to flash the fpga. """
 
-        # TODO: we checkout a specific version of aws-fpga here, in case upstream
-        # master is bumped. But now we have to remember to change AWS_FPGA_FIRESIM_UPSTREAM_VERSION
-        # when we bump our stuff. Need a better way to do this.
-        AWS_FPGA_FIRESIM_UPSTREAM_VERSION = "6c707ab4a26c2766b916dad9d40727266fa0e4ef"
-        self.instance_logger("""Installing AWS FPGA SDK on remote nodes. Upstream hash: {}""".format(AWS_FPGA_FIRESIM_UPSTREAM_VERSION))
+        with cd("../"):
```

Comment:
Nit: would it not be safer to make this absolute (since we're already using absolute paths)?

Reply:
Currently, the only absolute paths used are on the run farm instances. This `local` call is done on the manager instance where you can't assume the location of the FireSim repo

Reply:
Ah, yes. Makes sense